rfc/rfc011-verifiable-credential.md

```diff
@@ -0,0 +1,215 @@
+
+### 4.2 Issuance & distribution
+
+A VC specification MUST specify how a VC is issued and who may issue it. It MUST specify which protocols and standards are used in obtaining the VC. 
+It MUST specify where it MAY be stored and if it's distributed to others in some way. It MUST specify any requirements for the holder.
+It MUST specify if VCs or other credentials are required in order to obtain the VC. It MUST specify the content-type `type` selector.
+
+### 4.3 Supported proofs
+
+It MUST specify which proof types are acceptable or if there's a limitation to certain proof types. 
+This also means that specifications SHOULD to be updated when new proof types are available.
+
+### 4.4 Trust
+
+It MUST list the requirements for when a VC is to be trusted. This can be any combination of configuration, other VCs and/or automatic mechanism.
+
+### 4.5 Revocation
+
+It MUST specify how a VC can be revoked by the issuer, or it MUST specify an expiration duration. It MAY refer to the default revocation mechanism stated below.
+
+### 4.5.1 Default revocation
+
+VCs that are issued by a Nuts DID can be revoked by publishing the following transaction on the network:
+
+```json
+{
+  "issuer": "did:nuts:t1DVVAs5fmNba8fdKoTSQNtiGcH49vicrkjZW2KRqpv",
+  "subject": "did:nuts:t1DVVAs5fmNba8fdKoTSQNtiGcH49vicrkjZW2KRqpv#6f91673b-afa9-4d26-9e0f-00d989943275",
+  "date": "2021-03-15T16:34:47.422436+01:00",
+  "proof": {
+    "type": "JsonWebSignature2020",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "did:nuts:t1DVVAs5fmNba8fdKoTSQNtiGcH49vicrkjZW2KRqpv#h22vbXHX7-lRd1qAJnU63liaehb9sAoBS7RavhvfgR8",
+    "created": "2021-03-15T16:34:47.422436+01:00",
+    "jws": "eyJhbGciOiJFUzI1NiIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..Rc7iK7wXabMx24ZNAFJIwxqpYGCdye0EmdOYwu5CO54pwEPNIQt-9qIvqEZ7ZBFcFUdhnNCvYkR8IDtFAM18Rw"
+  }
+}
+```
+
+Such a revocation transaction has the following requirements:
+
+* the **issuer** MUST match the DID as the `issuer` field of the VC.
+* the **subject** MUST match the `id` field of the VC.
+* a **reason** MAY be filled with a revocation reason.
+* the **date** MUST provide the date in rfc3339 format. From this moment in time the VC is revoked.
+* the **proof** MUST be a `JsonWebSignature2020` proof.
+
+The transaction MUST be published on the Nuts network. It MAY use the `verificationMethod` for the transaction.
+The content-type is `application/vc+json;type=revocation`
+
+The signature is calculated as stated in §3.1.1.
+
+### 4.6 Use cases
+
+It MUST specify where the VC SHOULD be used: as requirement for other VCs, in the oauth flow according to [RFC003](rfc003-oauth2-authorization.md) or any other use case.
+If a VC can be used in a Verifiable Presentation (VP), it MUST specify if additional VCs are to be expected in the VP.
+It SHOULD describe the use case well enough so any implementation can take appropriate measures, such as indexing certain fields.
```

Comment:
"can take appropriate measures" for what? Security/performance/optimization/quality/pretty-ness? I think you mean something like "for optimizing querying/checking"

Reply:
yes that